include/my_time.h

```diff
@@ -244,9 +244,12 @@ static inline void my_time_trunc(MYSQL_TIME *ltime, uint decimals)
       !ltime->hour && !ltime->minute && !ltime->second)
     ltime->neg= FALSE;
 }
+#ifdef __WIN__
```

Comment:
`_WIN32` wins the popularity contest in 10.5 over `__WIN__`, so perhaps we would want to use that.